src/server/pps/server/api_server.go

```diff
@@ -600,6 +600,9 @@ func (a *apiServer) validatePipeline(ctx context.Context, pipelineInfo *pps.Pipe
 	if pipelineInfo.OutputBranch == "" {
 		return fmt.Errorf("pipeline needs to specify an output branch")
 	}
+	if pipelineInfo.ParallelismSpec != nil && pipelineInfo.ParallelismSpec.Constant == 0 && pipelineInfo.ParallelismSpec.Coefficient == 0 {
+		return fmt.Errorf("cannot specify 0 for both constant and coefficient in parallelism spec")
+	}
```

Comment:
Shouldn't this really check if `Strategy == "CONSTANT" && Constant == 0`? Joey hit this error because he was unwittingly setting `Strategy == "COEFFICIENT"` but only setting `Constant` to a nonzero number.